mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ContainerEventHandlerAdapterBuilder.java

```diff
@@ -1,8 +1,9 @@
 package org.wildfly.extension.mod_cluster;
 
+import org.jboss.as.controller.OperationContext;
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceTarget;
 
 public interface ContainerEventHandlerAdapterBuilder {
-    ServiceBuilder<?> build(ServiceTarget target, String connector, int statusInterval);
+    ServiceBuilder<?> build(ServiceTarget target, OperationContext context, String connector, int statusInterval);
```

Comment:
Actually, I should think you can change it to:
`ServiceBuilder<?> build(CapabilityServiceTarget target, String connector, int statusInterval);`